m4/sage_spkg_configure.m4

```diff
@@ -185,22 +185,40 @@ AC_DEFUN([SAGE_SPKG_CONFIGURE], [
 
 # SYNOPSIS
 #
-#   SAGE_SPKG_DEPCHECK(PACKAGE-DEPENDENCIES, FURTHER-CHECK)
-#                                 $1              $2
+#   SAGE_SPKG_DEPCHECK(PACKAGE-DEPENDENCIES, FURTHER-CHECK, [FAIL= [sage_spkg_install_]SPKG_NAME=yes])
+#                                 $1              $2                $3
 #
 # DESCRIPTION
 #     *** to be called from SAGE_SPKG_CONFIGURE* ***
 #     check for space-separated list of package dependencies $1 of package SPKG_NAME
 #     do $2 if successful
```

Comment:
likewise here, the "successful" is unclear.

Reply:
see https://github.com/sagemath/sage/pull/36813/commits/b5e2e6562b76913aa646e11de158b0880af93f40